src/test/java/com/newestuser/imen/ImenApplicationTests.java

```diff
@@ -9,8 +9,8 @@
 @SpringBootTest
 public class ImenApplicationTests {
 
-	@Test
-	public void contextLoads() {
-	}
+  @Test
```

Comment:
I am not sure that this is the correct indentation. We need to make sure that the tests are also scanned by checkstyle.

Reply:
I have corrected the pom.xml file. The tests  are also scanned by the checkstyle .